### PR TITLE
Add vars templates for Stream service and allow to deploy them

### PR DESCRIPTION
**Pull request by jpopelka**
by specifying `SERVICE` environment variable.

E.g.
 `SERVICE=stream DEPLOYMENT=dev make deploy`.

If you have a better name for the SERVICE (env. var.) / service (ansible var.), let me know.
EDIT: Another name might be PROJECT - in a sense of kb8/openshift project.

- [X] [update secrets repo](https://gitlab.cee.redhat.com/user-cont/packit-service/-/merge_requests/146)
- [x] create TLS certificates

### CODE REVIEW

**Comment by jpopelka** [comment]
recheck

File: README.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 ## tl;dr How to deploy
 
 1. Obtain all the necessary [secrets](/secrets/README.md).
-2. in [vars](vars/) copy `{deployment}_template.yml` to `{deployment}.yml` where `{deployment}` is one of `prod`, `stg` or `dev` and fill in values
+2. in [vars/packit](vars/packit/) copy `{deployment}_template.yml` to `{deployment}.yml` where `{deployment}` is one of `prod`, `stg` or `dev` and fill in values
 3. `dnf install ansible origin-clients python3-openshift`
 4. `DEPLOYMENT={deployment} make deploy`
```
Comment (lbarcziova) [inline README.md:8]: can we make the README now more general starting with changing the first line to `Ansible playbooks and scripts for deploying Packit service and Stream service to Openshift` and changing the 2. step here to `vars/packit or vars/stream` and in 4. step `SERVICE={service} DEPLOYMENT={deployment} make deploy`?

**Review by lbarcziova — APPROVED** [review]
LGTM!

File: README.md (hunk 1 above)
Comment (jpopelka) [reply README.md]: updated, thanks

**Review by lachmanfrantisek — APPROVED** [review]
I can't say anything against. Thanks!

**Pull request merged**